Analytics/EntityStoreTools/Common/Contracts/AxViewMetadata.cs

```diff
@@ -0,0 +1,21 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
Same here: this file is the same as in `Analytics/Common`. Isn't possible to reuse it to prevent this duplication?